lib/src/deprecations.ts

```diff
@@ -15,12 +15,83 @@ export {Deprecation, DeprecationOrId, DeprecationStatus} from './vendor/sass';
 export function getDeprecationIds(
   arr: (DeprecationOrId | Version)[]
 ): string[] {
-  return arr.flatMap(item => {
+  return arr.map(item => {
     if (item instanceof Version) {
-      return arr.map(item => item.toString());
+      return item.toString();
     } else if (typeof item === 'string') {
       return item;
     }
     return item.id;
   });
 }
+
+/**
+ * Shorthand for the subset of options related to deprecations.
+ */
+export type DeprecationOptions = {
```

Comment:
Consider using [`Pick<>`](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys) to derive this from the options type.